database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,12 +15,13 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('temporary_name')->nullable();
```

Comment:
My purpose for this seperate project is to transfer all of the code base and make our code more reusable. Since I'm transferring all of the codes from the new repository. I think I don't need to create a new migration for this.